pyproject.toml

```diff
@@ -85,7 +85,7 @@ include = [
     "/README.md",
     "/requirements-test.txt"
 ]
-artefacts = [
+artifacts = [
```

Comment:
This was a typo, so these were not included before. Are they needed?

Reply:
Are you SERIOUS? That's frustrating (and definitely my British fault). We certainly want these in our sdists.

Reply:
There are two options - either Hatchling could provide a json-schema, and then validate-pyproject would be able to validate the `tool.hatch` section, and/or hatchling could gain a `strict-config` setting, which I don't see present. (Scikit-build-core will have a schema in 0.5). Not sure if @ofek wants to work on it or would accept work toward it (it really depends on the backend settings implementation - it was trivial with scikit-build-core, because it was declarative dataclass-based).

Reply:
A strict option would be doable but a schema-based implementation would be a large effort

Reply:
I'm resolving this as I've corrected it in `main`.